client/domain/CodeEvalResultObjectFactorySpec.js

```diff
@@ -45,21 +46,28 @@ describe('CodeEvalResultObjectFactory', function() {
   describe('getCorrectnessTestResults', function() {
     it('should correctly get correctness test results', function() {
       expect(codeEvalResult.getCorrectnessTestResults())
-      .toMatch(CORRECTNESS_TEST_RESULTS);
+        .toEqual(CORRECTNESS_TEST_RESULTS);
+    });
+  });
+
+  describe('getLastTaskResults', function() {
+    it('should correctly get last task results', function() {
+      expect(codeEvalResult.getLastTaskResults())
+        .toEqual(LAST_TASK_RESULTS);
```

Comment:
Wait, hang on. LAST_TASK_RESULTS should be local to this function since it's just an expected value. It doesn't need to be declared globally.